cpp/src/bindings.cpp

```diff
@@ -48,6 +48,10 @@ PYBIND11_MODULE(_pypowsybl, m) {
 
     m.def("create_eurostag_tutorial_example1_network", &pypowsybl::createEurostagTutorialExample1Network, "Create an Eurostag tutorial example 1 network");
 
+    m.def("create_eurostag_tutorial_example1_network_with_fixed_current_limits", &pypowsybl::createEurostagTutorialExample1NetworkWithFixedCurrentLimits,
```

Comment:
If this network has only current limits in addition to base one, maybe we could just replace the create_eurostag_tutorial_example1 (but let's call it as before simply create_eurostag_tutorial_example1) and use in Java the one with current limits?

Reply:
I agree. I just need to check if tests are not broken.